octavia_proxy/api/drivers/elbv3/driver.py

```diff
@@ -39,18 +39,24 @@ def get_supported_availability_zone_metadata(self):
         return {"eu-nl-01": "The compute availability zone to use for "
                             "this loadbalancer."}
 
-    def _normalize_lb(self, lb):
-        return self._normalize_tags(lb)
-
-    def _normalize_tags(self, lb):
-        tags = []
-        otc_tags = lb.tags
+    def _normalize_tags(self, resource):
+        otc_tags = resource.tags
         if otc_tags:
             tags = []
-            for k, v in otc_tags:
-                tags.append('%s=%s' % (k, v))
-            lb.tags = tags
-        return lb
+            for tag in otc_tags:
+                tags.append('%s=%s' % (tag['key'], tag['value']))
+            resource.tags = tags
+        return resource
+
+    def _resource_tags(self, tags):
+        result = []
+        for tag in tags:
+            tag = tag.split('=')
```

Comment:
User might really pass `white` as a tag, thus split fails.

Reply:
https://prnt.sc/1ttnkpt

Reply:
i dont get, why it fails should fail

Reply:
```
>>> s = "white".split("=")
>>> s[1]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```

Reply:
done